weave/bin/modules/annotations/search.py

```
@@ -0,0 +1,248 @@
+        
+        client = self.get_elasticsearch_client()
+        doc_id = getattr(self, config.id_field)
+        
+        try:
+            client.delete(
+                index=config.index_name,
+                id=doc_id
+            )
+        except Exception:
+            # Document might not exist, ignore
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)